core/runtime.rs

```diff
@@ -505,34 +505,25 @@ impl JsRuntime {
     Ok(())
   }
 
-  /// Grab a Global handle to a function returned by the given expression
-  fn grab_fn(
-    scope: &mut v8::HandleScope,
-    code: &str,
-  ) -> v8::Global<v8::Function> {
-    let code = v8::String::new(scope, code).unwrap();
-    let script = v8::Script::compile(scope, code, None).unwrap();
-    let v8_value = script.run(scope).unwrap();
-    let cb = v8::Local::<v8::Function>::try_from(v8_value).unwrap();
-    v8::Global::new(scope, cb)
-  }
-
-  fn grab_obj<'s>(
+  /// Grab a Global handle to a v8 value returned by the expression
+  fn grab_js<'s, T>(
     scope: &mut v8::HandleScope<'s>,
     code: &str,
-  ) -> v8::Local<'s, v8::Object> {
-    let scope = &mut v8::EscapableHandleScope::new(scope);
+  ) -> v8::Global<T>
+  where
+    v8::Local<'s, T>: TryFrom<v8::Local<'s, v8::Value>, Error = v8::DataError>,
+  {
     let code = v8::String::new(scope, code).unwrap();
     let script = v8::Script::compile(scope, code, None).unwrap();
```

Comment:
With only two callers, this hardly matters, but...

Seems a bit excessive to compile a script for this.
A generic-enough alternative might be 
```rust
fn get_deno_core_property<T>(
  scope: &mut v8::HandleScope<'s>, 
  name: &str // e.g. "ops"
) -> v8::Local<'s, T>
where
    v8::Local<'s, T>: TryFrom<v8::Local<'s, v8::Value>, Error = v8::DataError>,
{ 
  // ... 
}
```

Reply:
Yeah, it's not called in hot paths so I've been valuing convenience over function.

A get or get+ensure & list of attr names would makes sense